opp_msgs/srv/GetPathSelection.srv

```diff
@@ -0,0 +1,12 @@
+# Request
+
+# The set of points from which the ROI
+# will be selected
+sensor_msgs/PointCloud2 input_cloud
+
+---
+
+# Response
+bool success
+string message
+int32[] cloud_indices
```

Comment:
This service message isn't built by the CMakeLists; it should be removed or the the CMakeLists should be amended